src/browser-lib/hubspot/forms/hubspotSubmitForm.test.ts

```diff
@@ -290,9 +290,9 @@ describe("hubspotSubmitForm", () => {
   describe("Network error", () => {
     beforeEach(() => {
       server.use(
-        rest.post(primaryFormEndpoint, (req, res) =>
+        http.post(primaryFormEndpoint, () =>
           // DEBUG this is now resulting in an OakError, so the following tests fail.
-          res.networkError("Failed to connect"),
+          HttpResponse.json("Failed to connect"),
```

Comment:
Here we want to tests "Network error" ... so there's not an HttpResponse ... we just throw an rather than responding.